server/app.js

```diff
@@ -1,7 +1,7 @@
-const express = require("express");
-//const csvToJson = require("csvtojson");
-const { Parser } = require("json2csv");
-const commissionRpt = require("./reports/commission/commission-report.js");
+import express from "express";
+import json2csvExports from "json2csv";
```

Comment:
There is something suspicious about this import statement. I should be able to write:
import {Parser} from "json2csv";
like before, but that generates errors like "The requested module 'json2csv' does not provide an export named 'Parser'" Maybe we'll learn why later in development. The code here does work, but I don't like the syntax.